### PR TITLE
Update i18n and helpUrl configuration 

### PR DESCRIPTION
**Pull request by qingdengyue**
1.add default.owner,logger,default.server,default.actives,default.client,default.connections  description.
  avoid error of provider page.
2.add [helpUrl](https://github.com/apache/incubator-dubbo-ops) .    #67

### CODE REVIEW

**Comment by qingdengyue** [comment]
#58
#66 
#55 
#46

File: dubbo-admin/src/main/resources/i18n/message.properties [hunk 1]
```diff
@@ -14,4 +14,30 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-
+dependencies=
```
Comment (htynkn) [inline dubbo-admin/src/main/resources/i18n/message.properties]: what's the purpose for this file change? my understanding is we need have different i18n information in message_{lang}.properties file

Comment (qingdengyue) [reply dubbo-admin/src/main/resources/i18n/message.properties]: Yes.It is.
message.properties‘s content is auto generated by IDEA Resource Bundle.I didn't have a attention. It had been removed.

**Comment by htynkn** [comment]
This pull request looks good to me
@ralf0131  I saw #74 was been merged. do you think this is any conflict with this pull request?

**Comment by nzomkxia** [comment]
@htynkn this pull request has no conflict with #74 and fix the invalid helpURL as well, you can merge this

**Pull request merged**